Documentation/coreos.rst

```diff
@@ -0,0 +1,356 @@
+.. _coreos_gs_guide:
+
+Getting Started Guide for CoreOS
+=====================
+
+This document serves as a guide to get Cilium up-and-running on CoreOS.
+
+If you haven't read the :ref:`intro` yet, we encourage you to do that first.
+
+For an introduction about how to setup Kubernetes with CoreOS, we recommend
```

Comment:
Will be great to cover in a sentence what this guide covers and for what we defer to the CoreOS guide here. The reader should know what to look for in that guide and when to return. Knowing my own reading pattern, I would never come back ;-)